tests/schedules_test.py

```diff
@@ -0,0 +1,374 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
As a py3 project, no need for `from __future__` and `coding: utf8` anymore

Reply:
oops. fixed.